app/src/services/subscribers/CensusEventSubscriberService.ts

```diff
@@ -24,6 +24,8 @@ import FacilityControlEvent from '../../handlers/census/events/FacilityControlEv
 
 @injectable()
 export default class CensusEventSubscriberService implements ServiceInterface {
+    public readonly bootPriority = 10;
```

Comment:
This potentially should come before the census socket service, as we want the handlers and events registered before the census socket begins connections?

Reply:
Census socket loads in the start method so maybe move the event subscriptions to the boot method?

Reply:
Seems reasonable enough